spec/react_on_rails/generators/install_generator_spec.rb

```diff
@@ -210,7 +211,7 @@
   end
 
   context "with missing files to trigger errors" do
-    it "GeneratorMessages has the missing file error" do
+    specify "GeneratorMessages has the missing file error" do
```

Comment:
it's the same thing as `it` basically, just reads more like english in this case because GeneratorMessages isn't the subject